landingzones/views_api.py

```diff
@@ -227,6 +234,62 @@ def perform_create(self, serializer):
             raise APIException('{}{}'.format(ex_msg, ex))
 
 
+class ZoneUpdateAPIView(
+    ZoneUpdateMixin, SODARAPIGenericProjectMixin, UpdateAPIView
+):
+    """
+    Update a landing zone.
+
+    **URL:** ``/landingzones/api/update/{LandingZone.sodar_uuid}``
+
+    **Methods:** ``PATCH``
```

Comment:
For consistency we should also support `PUT`, which requires both the `description` and `user_message` fields to be present.